specification/attestation/data-plane/Microsoft.Attestation/preview/2018-09-01-preview/attestation.json

```diff
@@ -258,17 +258,163 @@
                 }
             }
         },
+        "/operations/policy/certificates": {
+            "get": {
+                "tags": [
+                         "Policy Management Certificates"
+                        ],
+                "summary": "Retrieves the set of certificates used to express policy for the current tenant.",
+                "operationId": "PolicyCertificates_Get",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "An RFC7519 JSON Web Token structure containing an RFC7517 JsonWebKeySet which contains the certificates used to validate policy changes",
+                        "schema": {
+                            "type": "string"
```

Comment:
Because you do not define a "produces", the default one "application/json" is used, this means the expected body bytes are surrounding with double quotes:
```
"ABCDEF"
```
If it's not what you expect, please add a consumes "text/plain" as you did for other operations

Reply:
Yes, this is what we expect.